### PR TITLE
Use https rather than http since TinyURL now support it

### LINKED ISSUE #1: Add support for HTTPS
http**s**://tinyurl.com/api-create.php?url=${encodeURIComponent(url)}

### PR DESCRIPTION
**Pull request by jkernech**
Fix #1

### CODE REVIEW

**Comment by jkernech** [comment]
As mentioned in #1, I am waiting for TinyUrl team feedback before merging

**Review by mrister — APPROVED** [review]

**Pull request merged**
